README.md

```diff
@@ -76,6 +76,21 @@ docker run -p 8080:8080 -v /your/local/assets/:/www/assets b4bz/homer:latest
 
 Default assets will be automatically installed in the `/www/assets` directory. Use `UID` and/or `GID` env var to change the assets owner (`docker run -e "UID=1000" -e "GID=1000" [...]`).
 
+### Using docker-compose
+
+```sh
+cd /path/to/docker-compose.yml
+docker-compose up -d
+```
+
+Default assets will be automatically installed in the `/www/assets` directory. Use `UID` and/or `GID` env var to change the assets owner :
```

Comment:
It would be great to add a bit more explanations, it's a bit misleading, as the `docker-compose.yml` must be edited (at least the `volumes: /your/local/assets/` part) to match your needs. Maybe explaining that a base / example to adapt ?

Thanks!